docs/source/quicktour.md

```diff
@@ -348,27 +348,39 @@ point in the script as shown above, and then, you should unwrap your model befor
 through the [`~Accelerator.prepare`] method, your model may have been placed inside a bigger model,
 which deals with the distributed training. This in turn means that saving your model state dictionary without taking
 any precaution will take that potential extra layer into account, and you will end up with weights you can't load back
-in your base model.
-
-This is why it's recommended to *unwrap* your model first. Here is an example:
+in your base model. The [`~Accelerator.save_model`] method will help you to achieve that. It will unwrap your model and save
+the model state dictionnary.
 
+Here is an example:
 ```
 accelerator.wait_for_everyone()
-unwrapped_model = accelerator.unwrap_model(model)
-accelerator.save(unwrapped_model.state_dict(), filename)
+accelerator.save_model(model, save_directory)
+```
+The [`~Accelerator.save_model`] method can also save a model into sharded checkpoints or with safetensors format.
+Here is an example: 
+
+```python
+accelerator.wait_for_everyone()
+accelerator.save_model(model, save_directory, max_shard_size="1GB", safe_serialization=True)
 ```
 
 If your script contains logic to load a checkpoint, we also recommend you load your weights in the unwrapped model
 (this is only useful if you use the load function after making your model go through
 [`~Accelerator.prepare`]). Here is an example:
 
-```
+```python
 unwrapped_model = accelerator.unwrap_model(model)
-unwrapped_model.load_state_dict(torch.load(filename))
+unwrapped_model.load_state_dict(torch.load(save_directory+"/pytorch_model.bin"))
```

Comment:
Here you should use `os.path.join` as this code won't work on Windows.

Reply:
great catch =)